src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/UpdateSubmission.java

```diff
@@ -53,6 +57,15 @@ public class UpdateSubmission {
     @JsonProperty(BENEFICIAL_OWNERS_FIELD)
     private BeneficialOwnersStatementType beneficialOwnerStatement;
 
+    @JsonProperty(CHANGE_BO_RELEVANT_PERIOD)
+    private RelevantStatementsType changeBORelevantPeriod;
```

Comment:
i think the class attributes should have the word Statement in them as a period can be different from a statement
eg changeBORelevantPeriodStatement
trusteeInvolvedRelevantPeriodStatement
changeBeneficiaryRelevantPeriodStatement